pwa/src/components/dashboardCard/DashboardCard.tsx

```diff
@@ -53,14 +53,7 @@ export const DashboardCard: React.FC<DashboardCardProps> = ({
       <div className={styles.statusTypeContainer}>
         <div>{_.upperFirst(type)}</div>
 
-        <div className={clsx(styles[getStatusColor(status ?? "no known status")])}>
-          <ToolTip tooltip="Status">
-            <Tag
-              icon={<FontAwesomeIcon icon={getStatusIcon(status ?? "no known status")} />}
-              label={status?.toString() ?? "no known status"}
-            />
-          </ToolTip>
-        </div>
+        {getStatusTag(status)}
```

Comment:
I think we will need to display what kind of error it is. See comment on `SourcesTemplate`